apps/fio/fio.pbs

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-FILESYSTEM=$1
```

Comment:
Can you please keep the FILESYSTEM as a parameter as this is a breaking changed. you can initialize it by default to the FILESYSTEM env variable. more generally speaking if you expect an env variable to be set make sur it's defined and exit on error if not.

Reply:
ok, thanks, I added the FILESYSTEM back as a parameter, setting the default to be $FILESYSTEM, so it can be set via environmental variable, PBS -v or command line arg.